Sources/Apollo/GraphQLOperation.swift

```diff
@@ -1,6 +1,7 @@
 public protocol GraphQLOperation: class {
   static var operationString: String { get }
   static var requestString: String { get }
+  static var operationIdentifier: String { get }
```

Comment:
One thought I had is to actually make `operationIdentifier`s in `apollo-codegen` *not* be controlled by the command-line `operation-ids-path` argument, and just *always* be generated. Then no default implementation would have to be provided. What do you think?